identity-server/src/Telemetry/Telemetry.cs

```diff
@@ -23,7 +23,8 @@ public static class Telemetry
     /// <summary>
     /// Service name used for the experimental non stable counters from Duende IdentityServer
     /// </summary>
-    public const string ServiceNameExperimental = "Duende.IdentityServer.Experimental";
+    [Obsolete("The experimental service name will be removed in a future version. The associated meter is now using the Duende.IdentityServer service name.")]
```

Comment:
I think we can just delete this entirely. If someone is using the old service constant, the fact that we're not using it anymore means that probably whatever they're doing with it is not semantically correct, even if it compiles with warnings. IMO a compilation failure in that case is better. Do you have a use case in mind where someone would want this constant, even though we don't have any counters built with it?

Reply:
Will drop it, agree with the suggestion. Thanks Joe!